recipes/samstats/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set org_name = "SAMstats" %}
+{% set name = "samstats" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ org_name }}-{{ version }}.tar.gz
+  sha256: 4d70604976b296ed6e0b7c561857aadc81324daabb983ca928300ac0f294025a
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - SAMstats = SAMstats.__init__:main
+    - SAMstatsParallel= SAMstatsParallel.__init__:main
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - multiprocess
+
+test:
+  imports:
+    - SAMstats
+    - SAMstatsParallel
+  commands:
+    - SAMstats --help
+    - SAMstatsParallel --help
+
+about:
+  home: The package home page
+  license: UNKNOWN
+  license_family: OTHER
+  license_file: ''
```

Comment:
This needs a license for us to legally redistribute it.

Reply:
thanks, license information updated (MIT)